src/ngraph/op/reshape.cpp

```diff
@@ -298,3 +362,9 @@ void op::v1::Reshape::generate_adjoints(autodiff::Adjoints& /* adjoints */,
 {
     throw ngraph_error("generate_adjoints not implemented for Reshape");
 }
+
+bool op::v1::Reshape::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs)
+{
```

Comment:
`v1::Reshape` is just a memcpy of all the bytes from the input to the output.
However, with `v1::Reshape` the output shape is an input which may have a -1 in it. If there is a -1, you need to figure out the actual length of that axis. In all cases, you need to set the actual output shape/element type before using it.